spec/prog/vnet/update_firewall_rules_spec.rb

```diff
@@ -67,14 +55,14 @@
     type ipv4_addr;
     flags interval;
     elements = {
-      10.0.0.0/26
+      10.0.0.0/32
     }
   }
 
   set private_ipv6_cidrs {
     type ipv6_addr
     flags interval
-    elements = { fd00::/64 }
+    elements = { fd00::1/128 }
```

Comment:
Interesting, what's going on here? May benefit from its own commit to explain this tiny change in constants

Reply:
I made it the very first commit. Please re-check